packages/cli/src/services/project-checker.ts

```diff
@@ -69,3 +70,60 @@ export async function checkResourceExists(name: string, placementDir: string, ex
     })
   }
 }
+
+export async function checkCurrentDirBoosterVersion(userAgent: string): Promise<void> {
+  return checkBoosterVersion(userAgent, process.cwd())
+}
+
+async function checkBoosterVersion(userAgent: string, projectPath: string): Promise<void> {
+  const projectVersion = await getBoosterVersion(projectPath)
+  const cliVersion = userAgent.split(' ')[0].split('/')[2]
+  await compareVersionsAndDisplayMessages(cliVersion, projectVersion)
+}
+
+async function getBoosterVersion(projectPath: string): Promise<string> {
+  const projectAbsolutePath = path.resolve(projectPath)
+  try {
+    const packageJsonContents = require(path.join(projectAbsolutePath, 'package.json'))
+    const version = packageJsonContents.dependencies['@boostercloud/framework-core']
+    const versionParts = version.replace('^', '').replace('.tgz', '').split('-')
+    return versionParts[versionParts.length-1]    
+  } catch (e) {
+    throw new Error(
+      `There was an error when recognizing the application. Make sure you are in the root path of a Booster project:\n${e.message}`
+    )
+  }
+}
+
+class HigherCliVersionError extends Error {
+  constructor(public cliVersion: string, public projectVersion: string, public section: string) {
+    super(`CLI version ${cliVersion} is higher than your project Booster version ${projectVersion} in the '${section}' section. Please upgrade your project dependencies.`)
+  }
+}
+
+class LowerCliVersionError extends Error {
+  constructor(public cliVersion: string, public projectVersion: string, public section: string) {
+    super(`CLI version ${cliVersion} is lower than your project Booster version ${projectVersion}. Please upgrade your @boostercloud/cli to the same version with npm`)
```

Comment:
Leveraging that we are writing a very nice error message (amazing, BTW), I would go further and add the command to upgrade the CLI version:
```suggestion
    super(`CLI version ${cliVersion} is lower than your project Booster version ${projectVersion}. Please upgrade your @boostercloud/cli to the same version with "npm install -g @boostercloud/cli@${projectVersion}"`)
```

Reply:
OK